### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by PastushenkoVik**

### CODE REVIEW

**Comment by PastushenkoVik** [comment]
- [UPDATE DEMO LINK](https://pastushenkovik.github.io/layout_product-cards/)
- [UPDATE TEST REPORT LINK](https://pastushenkovik.github.io/layout_product-cards/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -9,11 +9,46 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
+<body class="body">
+  <div class="goods-card" data-qa="card">
+    <div class="goods-img">
+      <img src="./images/imac.jpeg" alt="GOODS">
```
Comment (irenhh) [inline src/index.html]: imagine you use screenreader and you can know what's on the page only with alt attribute, will you recognize what `GOODS` mean on this page?
P.S. for alt text use only lowercase

File: src/index.html [hunk 2]
```diff
@@ -9,11 +9,46 @@
...
+    </div>
+    <div class="goods-info">
+      <div class="goods-name">
```
Comment (irenhh) [inline src/index.html]: here is better to use heading tag instead of just div

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
+.body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  color: #000;
```
Comment (irenhh) [inline src/style.css]: this color is by default

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.goods-card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
```
Comment (irenhh) [inline src/style.css]: you don't need to set `background #fff` here

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+.goods-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.goods-id {
+  font-weight: normal;
```
Comment (irenhh) [inline src/style.css]: this font-weight is by default here

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+  align-items: center;
+}
+
+.star-cell img {
+  width: 100%;
+  height: 100%;
+}
+
+.goods-response {
+  font-weight: normal;
```
Comment (irenhh) [inline src/style.css]: font-weight isn't needed here

File: src/index.html [hunk 7]
```diff
@@ -9,11 +9,46 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
+<body class="body">
+  <div class="goods-card" data-qa="card">
+    <div class="goods-img">
+      <img src="./images/imac.jpeg" alt="Моноблок APPLE A1419 iMac 27&#x201c; Retina 5K (MNED2UA/A)">
```
Comment (irenhh) [inline src/index.html]: for alt 'apple imac' would be enough, more information available in the text below,
and also for alt attribute only lowercase letters are used

**Pull request closed without merging**